sources/packages/web/src/views/aest/student/applicationDetails/RestrictionsManagement.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <full-page-container layout-template="centered">
+    <template #header>
+      <header-navigator
+        title="Application history"
+        :routeLocation="{
+          name: AESTRoutesConst.STUDENT_APPLICATIONS,
+          params: { studentId },
+        }"
+        subTitle="Application Restriction Management"
+      />
+    </template>
+    <history-bypassed-restrictions
+      class="mb-5"
+      :applicationId="applicationId"
+      @viewBypassDetails="goToViewBypassDetails"
+      @viewRemoveBypass="gotToViewRemoveBypass"
+      :key="historyKey"
+    />
+  </full-page-container>
```

Comment:
Are we considering to display Date Submitted here?

![image](https://github.com/user-attachments/assets/da56f240-e89b-4132-ac17-a8c41ab60851)

Reply:
Added.